pythonAPIClient/client.py

```diff
@@ -298,21 +297,164 @@ def usage_by_client(self, clientID, owner):
         return response.json()['api_usage']
 
     # TODO: add a camera to database
-    def add_camera(self, camera):
+    def add_camera(self, camera_type=None, is_active_image=None, is_active_video=None, ip=None, snapshot_url=None,
```

Comment:
why is add cam changed here same i saw that the definition of the update cam was changed in add cam any special reason?

Reply:
This was just to match the parameters in the add_camera branch